config.m4

```diff
@@ -70,6 +70,16 @@ if test "$PHP_SMBCLIENT" != "no"; then
     -lsmbclient
   ])
 
+  dnl Check for smbc_setOptionProtocols;
+  dnl seems missing on Alpine
```

Comment:
It's not missing on Alpine Linux in general, it's s missing in old libsmbclient versions.

It seems that `smbc_setOptionProtocols` has been introduced in libsmbclient 4.10.0, and old Alpine versions (3.9 and older) use an older libsmbclient version (see [`libsmbclient.h` of v4.9.18](https://github.com/samba-team/samba/blob/samba-4.9.18/source3/include/libsmbclient.h) vs [`libsmbclient.h` of v4.10.0](https://github.com/samba-team/samba/blob/samba-4.10.0/source3/include/libsmbclient.h#L851))